inference-engine/ie_bridges/java/sample/face_detection_sample_async/Main.java

```diff
@@ -0,0 +1,246 @@
+import org.opencv.core.*;
+import org.opencv.imgcodecs.*;
+import org.opencv.videoio.*;
+import org.opencv.videoio.VideoWriter;
+import org.opencv.core.Size;
+import org.opencv.imgproc.Imgproc;
+
+import java.util.LinkedList;
+import java.util.Vector;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.Map;
+import java.util.Queue;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.intel.openvino.*;
+
+public class Main {
+ 
+    public static Blob imageToBlob(Mat image) {
+        if (buff == null) 
+            buff = new byte[(int) (image.total() * image.channels())];
+
+        image.get(0, 0, buff);
+
+        int[] dimsArr = {1, image.channels(), image.height(), image.width()};
+        TensorDesc tDesc = new TensorDesc(Precision.U8, dimsArr, Layout.NHWC);
+
+        return new Blob(tDesc, buff);
```

Comment:
We need to exclude memory allocations for infer requests input blobs - so just create blobs once and then set them as inputs with copied frame.